hack/examples/patches/nutanix/initialize-variables.yaml

```diff
@@ -17,26 +17,26 @@
             strategy:
```

Comment:
These are coming from the initialize variables patch elsewhere so please just update `"/spec/topology/variables0/value/nutanix"`.

Reply:
updated. thanks